modules/ossm-control-plane-profiles.adoc

```diff
@@ -0,0 +1,87 @@
+// Module included in the following assemblies:
+//
+// * service_mesh/v2x/prepare-to-deploy-applications-ossm.adoc
+
+[id="ossm-control-plane-profiles_{context}"]
+= Creating control plane profiles
+
+You can create reusable configurations with `ServiceMeshControlPlane` profiles. Individual users can extend the profiles they create with their own configurations. Profiles can also inherit configuration information from other profiles. For example, you can create an accounting control plane for the accounting team and a marketing control plane for the marketing team. If you create a development template and a production template, members of the marketing team and the accounting team can extend the development and production profiles with team specific customization.
+
+When you configure control plane profiles, which follow the same syntax as the `ServiceMeshControlPlane`, users inherit settings in a hierarchical fashion. The Operator is delivered with a `default` template with default settings for {ProductName}. 
```

Comment:
should this be profile? 
The Operator is delivered with a `default` profile with default settings for {ProductName}.